src/servers/tests/mod.rs

```diff
@@ -54,9 +54,18 @@ impl DummyInstance {
 
 #[async_trait]
 impl SqlQueryHandler for DummyInstance {
-    async fn do_query(&self, query: &str, query_ctx: QueryContextRef) -> Result<Output> {
+    async fn do_query(&self, query: &str, query_ctx: QueryContextRef) -> Vec<Result<Output>> {
         let plan = self.query_engine.sql_to_plan(query, query_ctx).unwrap();
-        Ok(self.query_engine.execute(&plan).await.unwrap())
+        let output = self.query_engine.execute(&plan).await.unwrap();
+        vec![Ok(output)]
+    }
+
+    async fn do_statement_query(
+        &self,
+        _stmt: sql::statements::statement::Statement,
+        _query_ctx: QueryContextRef,
+    ) -> Result<Output> {
+        unreachable!()
```

Comment:
```suggestion
        unimplemented!()
```

`unimplemented!()` like other dummy instances